dask_image/ndfilters/_conv.py

```diff
@@ -17,7 +17,7 @@ def convolve(image,
     depth, boundary = _utils._get_depth_boundary(image.ndim, depth, "none")
 
     result = image.map_overlap(
-        scipy.ndimage.filters.convolve,
+        convolve_dispatch(image),
```

Comment:
This is a little bit weird because we need to pass an argument to the dispatcher (so it can check the array type) but it returns a bare function

Reply:
It does seem a bit simpler overall

Reply:
That's a good point. Made a couple suggestions below that may help. Though please let me know if you already tried them and ran into issue.

Reply:
We may also need to make this change as well. Sorry for missing that before.

```suggestion
        convolve_dispatch,
```

Reply:
> We may also need to make this change as well. Sorry for missing that before.

This makes more sense to me now, thanks!